pages/tools/components/Sidebar.tsx

```diff
@@ -79,8 +91,14 @@ export default function Sidebar({
         <SearchBar transform={transform} />
         {filters.map(({ label, accessorKey }) => {
           const checkedValues = transform[accessorKey as keyof Transform] || [];
+          const IconComponent =
+            filterIcons[accessorKey as keyof typeof filterIcons];
           return (
-            <DropdownMenu key={accessorKey} label={label} icon={<FilterIcon />}>
+            <DropdownMenu
+              key={accessorKey}
+              label={label}
+              icon={IconComponent && <IconComponent />}
```

Comment:
This check is irrelevant as IconComponent will always be present 
```suggestion
              icon={<IconComponent />}
```